src/lib/viewers/image/ImageLoader.js

```diff
@@ -65,6 +72,31 @@ class ImageLoader extends AssetLoader {
             return viewer.REP === entry.representation;
         });
     }
+
+    /**
+     * @inheritdoc
+     */
+    determineViewer(file, disabledViewers = []) {
+        return this.viewers.find(viewer => {
```

Comment:
Can we avoid duplicating the logic from AssetLoader? Can we call `super.determineViewer` and then further filter the list as needed?

Reply:
We could probably do something like that if `determineViewer()` returned a set of viewers rather than a single viewer. Then we could call `super.determineViewer()` and then further filter the results. Happy to make that change if you'd like, but it'll increase the scope of this change.

Reply:
Alternatively, I could probably create a new method, `AssetLoader.determineViewerSet()` that returns a set of viewers and then do this:

```
// AssetLoader
determineViewer(file, disabledViewers = []) {
  const viewers = determineViewerSet(file, disabledViewers);
  if (viewers.length > 0) {
    return viewers[0];
  }

  return undefined;
}
```

and then in `ImageLoader`:

```
// ImageLoader
determineViewer(file, disabledViewers = []) {
  const viewers = super.determineViewerSet(file, disabledViewers);
  // perform further filtering...
}
```